packit/local_project.py

```diff
@@ -131,13 +132,23 @@ def refresh_the_arguments(self):
             )
 
     def _is_url(self, path_or_url):
+        try:
```

Comment:
There are multiple issues in these 5 lines:
1. It won't work since you have to call _parse_url like this: `self._parse_url`.
2. doing `except:` is dangerous because you are catching every exception possible: you definitely don't want that
3. Since the function just calls to _parse_url, I don't it's needs: I feel like they should indeed by merged or restructured